werkit/aws_lambda/test_integration.py

```diff
@@ -86,6 +86,7 @@ def test_integration_success(tmpdir):
         assert isinstance(data["start_timestamp"], float)
 
         results = data["results"]
+        print(results)
```

Comment:
You probably want to remove this rogue print invocation.

Reply:
I think it's probably useful to keep it. When these fail this is basically the starting point, and pytest suppresses the output by default. I could add a comment if it seems helpful.